### DISCUSSION TITLE
Max Protection DNS-over-HTTPS via Policy?

### DISCUSSION THREAD

**Discussion by wizardberry** [mozilla/policy-templates#1062] [Ideas]
It looks like the DOH policy settings only set Firefox to Increased Protection, rather than Max Protection.  Increased seems to allow fallback to system settings, while Max Protection will bail if the DOH fails.  In some cases, it is preferable to force the queries through DOH and never allow fallback to system.

**Comment by mkaply** [comment]
Thanks for pointing this out. I'm going add a Fallback setting so you can set the equivalent of Max.

**Comment by mkaply** [comment]
Fallback was added.
